### DISCUSSION TITLE
How i can cache Select options?

### DISCUSSION THREAD

**Discussion by prksv** [filamentphp/filament#11596] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

I'm using a third party API to get options. My select is dependent on the selection of the previous input. How can I cache them so that a new request is not sent when rerendering?
![Без имени](https://github.com/filamentphp/filament/assets/94474856/2bd3fcf1-3c00-49e7-9ea2-4ce067854da4)

**Comment by mpskovvang** [comment]
I'm having a similar issue but with internal SQL/Eloquent queries.

My form has several live components affecting other parts of the form. Unfortunately, this reactivity triggers an unnecessary refresh of multiple `Select` component options.

It would be great to have the option to cache/persist loaded options during a re-render on the client side.

My workaround is to use a short cache on the server side for preloaded options.

**Comment by e2d2-dev** [comment]
using the cache facade?!?

        Cache::remember('stuff', 60, function () {
            return MyApiCall::make(); 
        });
